### ISSUE TITLE
Old Firmware - New message

### ISSUE THREAD

**Issue by guardianbs** [AlexxIT/XiaomiGateway3#1340]

**Comment by guardianbs** [comment]
I have been using your excellent integration for a very long time without issue to the point where my HA depends upon it.

I have 2 ZNDMWG03LM lumi.gateway.mgl03 with firmware 1.4.7_0115 with 47 Devices, 194 entities attached which have performed without fault up to version 3.3.6.  I have just updated to version 4.0.4 and now receive this message for the first time

{'msg': "Unsupported firmware: ........, 'model': 'lumi.gateway.mgl03', ........, 'version': '1.4.7_0115'}"}

I have read the Documentation on supported firmware and can see that this version is no longer supported. I have not had the need to read the Documentation for a long time and I am sure that this version used to be supported. I did not see anything in the release notes for version 4 to say that this version would no longer be supported.

The MiHome app is only offering an firmware update to version 1.4.7_0160 which suggests that this is the latest version that the hardware will support.

I have not had long to properly test version 4.0.4 but all devices still appear to be working.

I'm just curious why this firmware version is no longer supported and that future changes to the integration may break the system or that there may be problems when it comes to adding new devices or re-pairing existing ones..
